Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.py

```diff
@@ -214,11 +248,50 @@ def build_filter_and_url_to_search_with(url_suffix: str, custom_filter: Optional
         request_data = json.loads(custom_filter)
     else:
         request_data = args_to_filter(arguments)
+
     request_data = {'filters': request_data} if 'filters' not in request_data.keys() else request_data
     return request_data, url_suffix
 
 
+def args_to_filter_close_alerts(alert_ids: Optional[List] = None,
+                                custom_filter: Optional[Union[str, dict]] = None,
+                                comment: Optional[str] = None,
+                                send_feedback: bool = False,
+                                feedback_text: Optional[str] = None,
+                                allow_contact: bool = False,
+                                contact_email: Optional[str] = None,
+                                reason: Optional[int] = None
+                                ):
+    request_data: Dict[str, Any] = {}
+
+    if custom_filter:
+        if isinstance(custom_filter, str):
+            request_data = json.loads(custom_filter)
+        else:
+            request_data = custom_filter
+    elif alert_ids:
+        request_data['filters'] = {
+            'id': {
+                'eq': alert_ids,
+            },
+        }
+    else:
+        raise DemistoException("Expecting at least one of the following arguments: alert_id, custom_filter.")
+
+    request_data['comment'] = comment
+    request_data['reason'] = reason
+    request_data['sendFeedback'] = send_feedback
+    request_data['feedbackText'] = feedback_text
+    request_data['allowContact'] = allow_contact
+    request_data['contactEmail'] = contact_email
```

Comment:
I believe this block should be located inside `elif alert_ids:`, so the `custom_filter` argument will truly override all other args